public/pages/Dashboard/Container/Dashboard.tsx

```diff
@@ -13,52 +13,42 @@
  * permissions and limitations under the License.
  */
 
-import React, { useEffect, useCallback } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { AppState } from '../../../redux/reducers';
 import { getDetectorList } from '../../../redux/reducers/ad';
 import { SORT_DIRECTION } from '../../../../server/utils/constants';
 import { EmptyDashboard } from '../Components/EmptyDashboard/EmptyDashboard';
-import {
-  EuiPageHeaderSection,
-  EuiTitle,
-  EuiPageHeader,
-  EuiPage,
-  EuiPageBody,
-  EuiLoadingSpinner,
-} from '@elastic/eui';
+import { EuiPage, EuiPageBody, EuiLoadingSpinner } from '@elastic/eui';
 import { SideBar } from '../Components/utils/SideBar';
 import { DashboardHeader } from '../Components/utils/DashboardHeader';
+import { DashboardOverview } from './DashboardOverview';
 
 export const Dashboard = () => {
   const dispatch = useDispatch();
 
-  const isLoading = useSelector((state: AppState) => state.ad.requesting);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Question: Could we be consistent , where to fetch and rely on data is it Redux or local states? I am seeing this entire place is mostly getting away with Redux and relying on local stats.

Reply:
Explained as above. Please share your thoughts on how to use same redux across multiple component on same page.